packages/Datadog.Unity/Runtime/DatadogPlatform.cs

```diff
@@ -32,4 +34,15 @@ internal interface IDatadogPlatform
 
         void ClearAllData();
     }
+
+    /// <summary>
+    /// The logging level for the DatadogSdk Core.
+    /// </summary>
+    public enum CoreLoggerLevel
+    {
+        Debug = 0,
+        Warn = 2,
```

Comment:
there is no `1`?

```suggestion
        Debug = 0,
        Warn = 2,
```

Reply:
Good catch, they should be sequential....